build_java.pl

```diff
@@ -162,6 +162,7 @@ sub setup_vars {
     }
     $jni_header_dir = "$dist_dir/private/jss/_jni";
 
+    $classpath = "-classpath /usr/share/java/apache-commons-codec.jar:/usr/share/java/commons-lang.jar:/usr/share/java/ldapjdk.jar:";
     if( $jce_jar ) {
         $classpath = "-classpath $jce_jar";
```

Comment:
This line will override the classpath. Is that intentional?

Reply:
In the master branch this line has been changed to append the classpath instead of replacing it.